injectproxy/enforce.go

```diff
@@ -111,18 +111,18 @@ func (ms Enforcer) EnforceNode(node parser.Node) error {
 	return nil
 }
 
-func (ms Enforcer) enforceMatchers(targets []*labels.Matcher) []*labels.Matcher {
```

Comment:
Do you mind adding quick commentary what this public method does (Public methods should be commented always with comment started as `// EnforceMatchers ...`

Reply:
sure!